src/Components/.editorconfig

```diff
@@ -0,0 +1,53 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
I think we're expected not to have per directory .editorconfig files

Reply:
Whelp. I'm tired of fighting with VS trying to put non-system stuff first. 

Is there a root `.editorconfig` somewhere and does it work?

Reply:
https://github.com/aspnet/AspNetCore/blob/master/.editorconfig < you have the 🔋

Reply:
It doesn't work then.

Reply:
If `.editorconfig` doesn't work, should we change our usings-ordering convention to "do whatever VS2019 does by default"?

Reply:
https://github.com/dotnet/roslyn/issues/33715

Reply:
We've done things in this style long before editorconfig. It's just annoying 😢

Reply:
This seems to be resolved by removing `:warning` from the rule and adding the editor config to the solution. I found a handy blog describing some of these rules (unofficial). https://kent-boogaart.com/blog/editorconfig-reference-for-c-developers